src/context/cart-context.js

```diff
@@ -0,0 +1,86 @@
+import axios from "axios";
+import { useState, useEffect, createContext, useContext } from "react";
+import { useWishlist } from "./wishlist-context";
+
+const CartContext = createContext();
+
+const CartProvider = ({ children }) => {
+	const [cartItems, setCartItems] = useState();
+	const { setWishlistProducts } = useWishlist();
+
+	const encodedToken =
```

Comment:
Instead of hard coding enocdedToken, You can use `localStorage` to save and fetch the token and then use it the axios calls.